packages/indexer/lib/index.js

```diff
@@ -137,6 +137,11 @@ class Indexer {
   } = {}) {
     let script = null;
     let scriptType = null;
+    if (fromBlock && !fromBlock.startsWith("0x")) {
```

Comment:
1. It's better if we test fromBlock is a string first, while TS handles the case when you are writing in TS, there is also the case where people use lumos from plain JavaScript
2. It might be better to split [this part](https://github.com/nervosnetwork/lumos/pull/99/files#diff-36c4b31e9f7cbb1fe191bfb2855bf31cR85) into a separate PR so the code can be shared.

Reply:
1. Add  `assertHexadecimal` util to check the value here;
2. Also add `assertHexString` util in this repo, and follow pr just need to rebase once it's merged into the develop branch.